docs/preview/02-Guidance/Service-to-service Correlation/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Service-to-service Correlation?"
+layout: default
+---
+
+# Service-to-service correlation
+The concept of service-to-service correlation is big and complex and spans multiple Arcus libraries. This user-guide will walk through all the available Arcus features that work together to facilitate service-to-service correlation in your application.
+
+## What is service-to-service correlation?
+Service-to-service correlation is a way to describe a relationship between different components where each 'component' represents a separated application. This could be, for example, an API call that results in an Azure Service Bus message that will be picked up by a message pump, or Azure Service Bus Azure Function that calls an additional API after receiving a message from a queue.
+
+Anywhere where one application calls another, that's where service-to-service correlation comes in. This way of correlation makes sure that the relationship of a single (business) transaction doesn't stop at the borders of one application or component, but continues in the other application. The end result is a clear overview of all the components involved.
+
+![Arcus service-to-service application map](/media/service-to-service-correlation-application-map.png)
+
+When using Arcus for service-to-service correlation, you'll see the this relationship in Application Insights in the [end-to-end transaction view](https://docs.microsoft.com/en-us/azure/azure-monitor/app/transaction-diagnostics):
+
+![Arcus service-to-service correlation relationship](/media/service-to-service-correlation-relationship.png)
+
+## Why should I use service-to-service correlation?
+Adding service-to-service correlation to your application adds many advantages:
+* ✔ Quicker spot performance bottlenecks and failures
+* ✔ Clear overview of used (Azure) resources (cost effective)
```

Comment:
Not completely sure what you mean by (cost effective). 
I haven't personally used this overview to get a view of costs.

Reply:
Yeah, I meant that people are getting a more clear overview of involved resources, and so, what resources are maybe not used and only increases the cost. But maybe that's too far-fetched 😄 .

Reply:
I'll remove the `(cost effective)` part.

Reply:
Done in https://github.com/arcus-azure/arcus.observability/pull/446/commits/7aca8a8e2d2160eae1f4cec0cd681c356358ad75